connect/runtime/src/main/java/org/apache/kafka/connect/runtime/AbstractHerder.java

```diff
@@ -245,34 +245,19 @@ public ConfigInfos validateConnectorConfig(Map<String, String> connectorConfig)
             ConfigDef basicConfigDef = (connector instanceof SourceConnector)
                                        ? SourceConnectorConfig.configDef()
                                        : SinkConnectorConfig.configDef();
-            Map<String, ConfigValue> validatedConnectorConfig = validateBasicConnectorConfig(
```

Comment:
Sure, if that's a superset of functionality then it should be called instead of just `validate`. I see the distributed herder's override.